### PR TITLE
Small fixups for git package

### PR DESCRIPTION
**Pull request by tolusha**
Signed-off-by: Anatoliy Bazko <abazko@redhat.com>

### What does this PR do?
Fix selenium tests in `git` package

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/10431

### CODE REVIEW

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/CommitFilesTest.java [hunk 1]
```diff
@@ -88,6 +90,7 @@ public void prepare() throws Exception {
     testProjectServiceClient.importProject(
         ws.getId(), Paths.get(resource.toURI()), PROJECT_NAME, ProjectTemplates.MAVEN_SPRING);
     ide.open(ws);
+    consoles.waitJDTLSProjectResolveFinishedMessage(PROJECT_NAME);
```
Comment (tsmaeder) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/CommitFilesTest.java:93]: Why do we need to wait for jdt.ls project import here?

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/RevertCommitTest.java [hunk 2]
```diff
@@ -72,10 +72,10 @@
 
   @BeforeClass
   public void prepare() throws Exception {
-    URL resource = getClass().getResource("/projects/git-pull-test");
     testUserPreferencesServiceClient.addGitCommitter(gitHubUsername, productUser.getEmail());
+    URL resource = getClass().getResource("/projects/git-pull-test");
```
Comment (tsmaeder) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/RevertCommitTest.java:76]: Random change or does this fix something?

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/RevertCommitTest.java [hunk 3]
```diff
@@ -72,10 +72,10 @@
...
     testProjectServiceClient.importProject(
-        ws.getId(), Paths.get(resource.toURI()), PROJECT_NAME, ProjectTemplates.MAVEN_SPRING);
+        ws.getId(), Paths.get(resource.toURI()), PROJECT_NAME, ProjectTemplates.NODE_JS);
```
Comment (tsmaeder) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/RevertCommitTest.java:78]: Again, why is this necessary?

**Review by svor — APPROVED** [review]

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/CommitFilesTest.java (hunk 1 above)
Comment (tolusha) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/CommitFilesTest.java:93]: Rename operation is perform during the test. It fails when jdt.ls isn't started.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/RevertCommitTest.java (hunk 2 above)
Comment (tolusha) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/RevertCommitTest.java:76]: Random change.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/RevertCommitTest.java (hunk 3 above)
Comment (tolusha) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/RevertCommitTest.java:78]: It isn't a maven project. 
If we keep maven template then jdt.ls seems continuously import the project.

**Review by tsmaeder — APPROVED** [review]
